js/charts/liveChart.js

```diff
@@ -14,8 +16,7 @@ $( document ).ready(function() {
   * TODO:  Add functions (like updateChart()) before $() call that would show
   *         or display different data to user. [frontend]
   *
-  * TODO:  Add Exchange name to websockets feed so app knows what title to
-  *        display.
+  * TODO:  Find a way to pass the required exchange in when a button is pressed
```

Comment:
It's what they should be able to do, but I haven't quite figured out how embedded javascript scripts work with html yet. It's probably a quick fix if you know what you are doing.